assembl/static2/js/app/components/common/fileUploader.jsx

```diff
@@ -116,9 +124,28 @@ class FileUploader extends React.Component<Object, FileUploaderProps, FileUpload
               }}
               alt="preview"
             />
+            <div className="preview-title">
+              <span className="assembl-icon-text-attachment" />
+              {imgTitle}
+            </div>
+            <OverlayTrigger placement="top" overlay={deleteThematicImageTooltip}>
+              <Button onClick={onDeleteClick} className="admin-icons">
+                <span className="assembl-icon-delete grey" />
+              </Button>
+            </OverlayTrigger>
           </div>
-        ) : null}
-        {!isImage && <div className="preview-title">{this.state.fileName}</div>}
+        ) : (
+          <div className={fileSrc && isImage ? 'preview' : 'hidden'}>
+            <img
+              src={fileSrc}
+              ref={(p) => {
+                this.preview = p;
```

Comment:
this.preview seems to be not used in the rest of the component

Reply:
@cedricmessiant is there a purpose for this ref ?

Reply:
I guess that there was a long time ago but it does not seem to be the case anymore. Can't find when that changed with blame and history so I guess we can remove it if everything seems fine

Reply:
Ok I'll remove it